tools/reviewer/job/sync/DiffConverter.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright 2018 The StartupOS Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.startupos.tools.reviewer.job.sync;
+
+import com.google.startupos.tools.reviewer.job.sync.GithubProtos.CommitRequest;
+import com.google.startupos.tools.reviewer.job.sync.GithubPullRequestProtos.CommitPointer;
+import com.google.startupos.tools.reviewer.job.sync.GithubPullRequestProtos.IssueComment;
+import com.google.startupos.tools.reviewer.job.sync.GithubPullRequestProtos.PullRequest;
+import com.google.startupos.tools.reviewer.job.sync.GithubPullRequestProtos.ReviewComment;
+import com.google.startupos.tools.reviewer.job.sync.GithubPullRequestProtos.User;
+import com.google.startupos.tools.reviewer.localserver.service.Protos;
+import com.google.startupos.tools.reviewer.localserver.service.Protos.Comment;
+import com.google.startupos.tools.reviewer.localserver.service.Protos.Diff;
+import com.google.startupos.tools.reviewer.localserver.service.Protos.GithubPr;
+import com.google.startupos.tools.reviewer.localserver.service.Protos.Thread;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DiffConverter {
```

Comment:
**oferb's comment:**
Please add a comment for what this class converts to/from.